src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -359,34 +359,34 @@ public ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, boo
                                                 activationAttributes);
         }
 
-        public ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, object?[]? activationAttributes)
+        public ObjectHandle? CreateInstanceFrom(string assemblyFile, string typeName, object?[]? activationAttributes)
         {
             return Activator.CreateInstanceFrom(assemblyFile, typeName, activationAttributes);
         }
 
-        public object? CreateInstanceFromAndUnwrap(string assemblyFile, string typeName)
+        public object CreateInstanceFromAndUnwrap(string assemblyFile, string typeName)
```

Comment:
Was this `?` removal intentional? Also in the following overrides of this API you also added a `!` in the return statement, but they still return nullable.